src/rust/organization-management/README.md

```diff
@@ -0,0 +1,17 @@
+## Organization Management
+
+### When making changes to the organization management service:
+
+- Table definitions can be found and altered in the migration file in the
+  `migrations` folder
```

Comment:
I'd reword this a bit; as it is, it implies there is a single file to interact with, and that developers should make changes to it, rather than creating new migrations.

It would also be good to say what the command to generate a new migration file is.